Revert "semaphore: temporarily disable the timedated test suite"

mrc0mmand opened the pull request:
This reverts commit ed9192bb55e23c00bf9ba810a949e7ffe1bbe959.

---

Reverts #17757. Merge after the issue described in [0] is resolved.

[0] https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=975010

mbiebl commented:
I've applied a workaround (https://salsa.debian.org/systemd-team/systemd/-/commit/e437da1e4fa0bcda592526fa26465bc9a79b3afb) in the debian/master branch and rebased the upstream-ci branch accordingly.

It should thus be safe to merge this PR now.

mrc0mmand commented:
> I've applied a workaround (https://salsa.debian.org/systemd-team/systemd/-/commit/e437da1e4fa0bcda592526fa26465bc9a79b3afb) in the debian/master branch and rebased the upstream-ci branch accordingly.
> 
> It should thus be safe to merge this PR now.

Thanks for the heads up! I rebased this PR on top of master & force pushed it to re-trigger Semaphore to confirm if it's indeed safe to merge.

mbiebl commented:
hm, looks like something is still at odds with Ubuntu :-/

bluca commented:
@mrc0mmand bionic-* are generally passing today, would you mind rebasing&pushing please?

bluca commented:
All green, party time!

yuwata commented:
@bluca We usually use "Rebase and merge" when PR contains only 1 commit, and use "Create a merge commit" when PR contains multiple commits.

bluca commented:
> @bluca We usually use "Rebase and merge" when PR contains only 1 commit, and use "Create a merge commit" when PR contains multiple commits.

Ah thanks for the heads-up, didn't know, will do that from now on.
